README.md

```diff
@@ -30,16 +30,12 @@ Some packages in otp-ui depend on sibling packages (e.g., `@opentripplanner/core
 
    > "@opentripplanner/package-to-test": **"file:../package-to-test"**
 
-2. In your source files, find and replace:
-
-   > opentripplanner/package-to-test/**lib**/types
-
-   with
+2. Run:
+   `yarn && yarn dev`
```

Comment:
Can we just add a predev script that is `"predev": "yarn"` so that this becomes simply `yarn dev`?

Reply:
Slightly inconsistent with the "Getting Started" section (there it says `yarn install` then `yarn dev`. If you update per @landonreed's comment above, then remember to update the "Getting Started" section.

Reply:
Fixed in 9befb32064eefdab6e9b8a0a41945a130872da10 and 1d369aea05797ecbe500da148cec00a439ec60e1.